opensquirrel/passes/exporter/cqasmv1_exporter.py

```diff
@@ -14,12 +15,13 @@ class _CQASMv1Creator(IRVisitor):
     # Precision used when writing out a float number
     FLOAT_PRECISION = 8
 
-    def __init__(self, register_manager: RegisterManager) -> None:
+    def __init__(self, register_manager: RegisterManager, barrier_links: list[list[Statement]]) -> None:
         self.register_manager = register_manager
         qubit_register_size = self.register_manager.get_qubit_register_size()
-        self.cqasmv1_string = "version 1.0\n\n{}\n\n".format(
+        self.cqasm_v1_string = "version 1.0\n\n{}\n\n".format(
```

Comment:
Should we just use `output`, as in the Writer? Simpler, consistent with the writer.

```suggestion
        self.output = "version 1.0\n\n{}\n\n".format(
```